addons/spreadsheet_dashboard_account/data/dashboards.xml

```diff
@@ -4,6 +4,8 @@
     <record id="dashboard_invoicing" model="spreadsheet.dashboard">
         <field name="name">Invoicing</field>
         <field name="spreadsheet_binary_data" type="base64" file="spreadsheet_dashboard_account/data/files/invoicing_dashboard.json"/>
+        <field name="main_data_model_ids" eval="[(4, ref('account.model_account_move'))]"/>
+        <field name="sample_dashboard_file_path">spreadsheet_dashboard_account/data/files/invoicing_sample_dashboard.json</field>
```

Comment:
Whyt is this not the same type as spreadsheet_binary_data ? Are binary fields worse in prformance in something ? (i'm not that familiar with the ORM)

Reply:
To prevent storing the sample dashboard json files in the filestore of each client database. This way we only store them once in the server 😉